django_socio_grpc/request_transformer/grpc_socio_proxy_context.py

```diff
@@ -30,14 +43,20 @@ def get_query_params(self, grpc_request_metadata):
     def build_absolute_uri(self):
         return "NYI"
 
+    def grpc_action_to_http_method_name(self, grpc_action):
+        for k, v in self.METHOD_MAP.items():
+            if grpc_action.lower().startswith(k.lower()):
```

Comment:
why a startswith here ?

Reply:
To match http method (GET, POST...) we need to call our service method (List..., Create...)